App/Inreach.php

```diff
@@ -217,7 +217,14 @@ function process_kml() {
 			Joe_Log::add(__('The KML response is invalid.', Joe_Config::get_item('plugin_text_domain')), 'error', 'empty_kml');
 		}
 	}
-	
+
+	private function truncateCoordinates($coordinates, $precision) {  
```

Comment:
Instead of passing the `$precision` as an argument I suggest moving reading the Setting inside the truncateCoordinates function itself.

I noticed you are rounding every value in the Array, but looking at the code I believe the coordinates accepts a third element - Elevation. Perhaps a check of the `$key` to avoid modifying non lat/lng value should be included?